env.ts

```diff
@@ -58,6 +58,10 @@ const envSchema = {
     desc:       "The URL of the search API.",
     devDefault: "https://discussions-rc.odl.mit.edu/api/v0/search/"
   }),
+  SENTRY_ENV: envalid.str({
+    desc:       "The environment for Sentry",
+    devDefault: "development"
```

Comment:
Because the default is a `devDefault`, an error will be thrown if
- `env` is imported, AND
- SENTRY_ENV is not defined, AND
- the environment is not development

Maybe that's desirable. We do want `SENTRY_ENV` to be defined after all.

But: Currently it is not defined in prod/rc, so we'll need to add it.